jira-status/action.yml

```
@@ -20,23 +20,35 @@ runs:
         fi
         issue_key=$(tr '[:lower:]' '[:upper:]' <<<"$issue_key")
         echo "issue_key=$issue_key" | tee -a $GITHUB_ENV
-    - name: Get issue info
-      id: issue
-      uses: gblearning/gbl-gha/jira-issue-get@v1
-      with:
-        jira_url: ${{ inputs.jira_url }}
-        jira_pat: ${{ inputs.jira_pat }}
-        issue_key: ${{ env.issue_key }}
-        issue_fields: status
-    - name: Set issue status
+    - name: Check whether issue belongs to active sprint
+      id: sprint
       shell: bash
       run: |
-        # Get issue current status
-        issue_status="$(jq -e -r '.fields.status.name' '${{ steps.issue.outputs.filename }}')"
-        echo "issue_status=$issue_status" | tee -a $GITHUB_ENV
+        # Get issue current sprint
+        http_code=$(curl -s \
+          -get \
+          -o sprint.json \
+          -H 'Authorization: Bearer ${{ inputs.jira_pat }}' \
+          -w'%{http_code}' \
+          -H'Content-Type: application/json' \
+          "${{ inputs.jira_url }}/rest/agile/1.0/issue/${{ env.issue_key }}?fields=status,sprint" \
+        )
+        if [ "$http_code" != "200" ]; then
+          cat 'sprint.json'
+          exit 1
+        fi
+        sprint_status="$(jq -e -r '.fields.sprint.state' sprint.json)"
+        issue_status="$(jq -e -r '.fields.status.name' sprint.json)"
+        if [ "$sprint_status" != "active" ]; then
+          cat 'sprint.json'
+          echo
+          echo "Not in active sprint"
```

Сделай вывод через команду ::error:: тут, чтобы не нужно было открывать полный лог когда ребята начнут жаловаться, что задачи не проходят проверку.

```suggestion
          echo "::error::Issue ${{ env.issue_key }} should be in an active sprint"
```